packages/capabilities/src/types.ts

```diff
@@ -38,6 +38,11 @@ export type StoreList = InferInvokedCapability<typeof list>
 // Top
 export type Top = InferInvokedCapability<typeof top>
 
+export type NoncePhrase = string
+export interface ShouldShowValidationNonce {
```

Comment:
How come `ShouldShow`?

```suggestion
export interface ValidationNonce {
```